xcconfig/common__target__ios.xcconfig

```diff
@@ -8,7 +8,7 @@
 
 // Architectures
 ARCHS = $(ARCHS_STANDARD)
-SDKROOT = iphoneos10.0
+SDKROOT = iphoneos
```

Comment:
I'm happy to remove this change if you like being more specific. This is just what Apple puts when you let them fix up your Xcode projects automatically.